README.md

```diff
@@ -150,7 +150,7 @@ Test files are located under `.template/spec` folder
 
 ## License
 
-This project is Copyright (c) 2014-2020 Nimble. It is free software,
+This project is Copyright (c) 2014-2021 Nimble. It is free software,
```

Comment:
To avoid this issue, I recommend updating all repositories with:

```
This project is Copyright (c) 2014 and onwards. It is free software,
```

Reply:
Sure @olivierobert I just saw that on the air-mailer repo 😁

Reply:
Updated in https://github.com/nimblehq/rails-templates/pull/266/commits/5fd80e4d171fe715714328d74c96af72d4f7e622